docs/user-guide-state-reporting.md

```diff
@@ -249,3 +249,28 @@ status:
 ```
 
 If you want to learn more about the `observedState` API, see [nmstate documentation](https://nmstate.github.io/).
+
+## Additional configuration
```

Comment:
small nit: It can be nice to add a note that the user need to remove manually all the nmstate-handler pods if he updates the configMap because the daemonset will not restart the pods on a change

Reply:
yep still work in progress there are some external operators that watch on the config map change and kill all the relevant pods but again is a separate project.

Reply:
Done